### ISSUE TITLE
Update GOVERNANCE to include sub project voting

### ISSUE THREAD

**Issue by caniszczyk** [cilium/community#82]
Outside of it being easier to find the governance of the project (e.g., I usually look for a GOVERNANCE.md file or a community repo), I'd like to ensure the voting is done in a transparent way.

As I understand it, all the governance is currently here: https://github.com/cilium/community

I see that it is being followed by adding maintainers, e.g., https://github.com/cilium/cilium/commit/3942bea6d5122c4367e9e7ee4495b744b742c7dd

There's been confusion in public how certain decisions have been made in the project which is an opportunity for clarity and growth imho: https://twitter.com/urlichsanais/status/1750513165148790918

For tetragon, do you have an example of where the vote happened for this? I recall that when the project was being moved to CNCF, some of the votes were happening in the Cilium Slack which isn't a best practice and we recommended recording everything on github and/or mailing list. If a vote can't be dug up, the CNCF requests that you correct things to reflect what is stated in your governance. If you need help on how to do this, look at other CNCF projects e.g., https://github.com/envoyproxy/envoy/blob/main/GOVERNANCE.md#adding-new-projects-to-the-envoyproxy-github-organization

If you want to use a tool that helps, we have https://gitvote.dev in CNCF that can help with voting.

**Comment by xmulligan** [comment]
Thanks for opening this issue, we are always trying to improve our governance process. We've gone through two governance reviews from TAG Contributor Strategy. See the latest one [here](https://github.com/cncf/tag-contributor-strategy/pull/496). During the first one last January, it was pointed out that having votes in slack was not a best practice because they are quickly lost to the sands of capitalism. Since then, we have started to record the votes in the git history so we have a record of them. Unfortunately, the Tetragon vote was done before we made this switch.

I'm going to be working with an LFX mentee in the next quarter to update and improve the governance process for Cilium https://github.com/cncf/mentoring/pull/1112. This will also be related to the work we will be doing for repo lifecycle https://github.com/cilium/community/issues/27. I'll add this issue as a part of what we will be working on. Once we have this in place, I'll work with the committers to make sure we have a recorded history of the votes and continue to do so going forward.
